src/components/calcite-switch/calcite-switch.scss

```diff
@@ -1,88 +1,88 @@
+  --calcite-switch-switched-track-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-handle-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-track-background: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-track-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-handle-border: theme("backgroundColor.brand-hover");
 }
 
 // sizes
 :host([scale="s"]) {
-  --calcite-switch-track-width: 28px;
-  --calcite-switch-track-height: 16px;
-  --calcite-switch-handle-size: 14px;
+  --calcite-switch-track-width: theme("spacing.6");
+  --calcite-switch-track-height: theme("spacing.4");
+  --calcite-switch-handle-size: theme("spacing.3");
 }
 
 :host([scale="m"]) {
   --calcite-switch-track-width: 36px;
-  --calcite-switch-track-height: 20px;
+  --calcite-switch-track-height: theme("spacing.5");
   --calcite-switch-handle-size: 18px;
 }
 
 :host([scale="l"]) {
-  --calcite-switch-track-width: 44px;
-  --calcite-switch-track-height: 24px;
-  --calcite-switch-handle-size: 22px;
+  --calcite-switch-track-width: theme("spacing.12");
+  --calcite-switch-track-height: theme("spacing.6");
+  --calcite-switch-handle-size: theme("spacing.5");
 }
 
 ::slotted(input) {
-  display: none;
+  @apply hidden;
 }
 
 :host {
-  display: inline-block;
-  position: relative;
-  width: auto;
-  cursor: pointer;
-  user-select: none;
-  vertical-align: middle;
+  @apply inline-block 
+    relative 
+    w-auto 
+    cursor-pointer 
+    select-none 
+    align-middle;
   tap-highlight-color: transparent;
 }
 
 :host([disabled]) {
-  opacity: var(--calcite-ui-opacity-disabled);
-  pointer-events: none;
-  cursor: default;
+  @apply opacity-disabled pointer-events-auto cursor-default;
 }
 // focus styles
 :host {
-  @include focus-style-base();
-  width: auto;
+  @apply focus-base w-auto;
 }
 
 :host(:focus) {
-  @include focus-style-outset();
+  @apply focus-outset;
 }
 
 .track {
-  position: relative;
-  display: inline-block;
-  vertical-align: top;
+  @apply relative 
+    inline-block 
+    align-top 
+    bg-foreground-2 
+    border 
+    border-solid 
+    border-color-2 
+    pointer-events-auto;
   width: var(--calcite-switch-track-width);
   height: var(--calcite-switch-track-height);
-  background-color: var(--calcite-ui-foreground-2);
   border-radius: 30px;
-  border: 1px solid var(--calcite-ui-border-2);
-  pointer-events: none;
   transition: all $transition;
 }
 
 .handle {
-  position: absolute;
-  display: block;
+  @apply absolute 
+    block 
+    right-auto 
+    bg-foreground-1 
+    border-2 
+    border-solid 
+    border-color-input;
   width: var(--calcite-switch-handle-size);
   height: var(--calcite-switch-handle-size);
   top: -1px;
   left: -1px;
-  right: auto;
-  background-color: var(--calcite-ui-foreground-1);
   border-radius: 30px;
-  border: 2px solid var(--calcite-ui-border-input);
-  pointer-events: none;
   transition: all $transition;
```

Comment:
It would be ideal if we could kill this variable.

Reply:
do we have a tailwind value I can use as a replacement?